### DISCUSSION TITLE
Black screen and endless spinner with 3 or more participant.

### DISCUSSION THREAD

**Discussion by stsmurfy** [edumeet/edumeet-client#179] [Q&A] [answered]
I've freshly fetched the latest source codes from the repository (edumeet-client, edumeet-media-node, edumeet-room-server). I launched them in the following sequence:

1. Media-Node
2. Room-Server
3. Client

As long as there are only two participants in the meeting, webcam sharing works fine, but as soon as a third person joins, everyone's video freezes (only black screen and a spinner). If the third person leaves, the videos of the other two return. What could I be doing wrong?

https://github.com/edumeet/edumeet-client/assets/25842875/f2b5cadc-62e9-4a25-97ae-ffb426f5a147

**Comment by havfo** [comment, marked as answer]
This means that your media-node is misconfigured somehow. It works with two people because that is P2P.

**Reply by stsmurfy** [reply]
Thanks. I found it. I've used the wrong IP as public IP when I started the media node. Mea culpa.
